waltz-test/src/main/python/waltz_ducktape/tests/validation/connection_interruption_test.py

```diff
@@ -0,0 +1,81 @@
+from waltz_ducktape.tests.produce_consume_validate import ProduceConsumeValidateTest
+from ducktape.mark.resource import cluster
+from ducktape.mark import parametrize
+from ducktape.cluster.cluster_spec import ClusterSpec
+from ducktape.utils.util import wait_until
+from time import sleep
+from random import randrange
+
+
+class ConnectionInterruptionTest(ProduceConsumeValidateTest):
+    """
+        Class for tests simulating network issues corresponding to Waltz Server and Waltz Storage nodes
+    """
+
+    MIN_CLUSTER_SPEC = ClusterSpec.from_list([
+        {'cpu': 1, 'mem': '1GB', 'disk': '25GB', 'additional_disks': {'/dev/sdb': '100GB'}, 'num_nodes': 3},
+        {'cpu': 1, 'mem': '3GB', 'disk': '15GB', 'num_nodes': 2},
+        {'cpu': 1, 'mem': '1GB', 'disk': '25GB', 'num_nodes': 1}])
+
+    def __init__(self, test_context):
+        super(ConnectionInterruptionTest, self).__init__(test_context=test_context)
+
+    @cluster(cluster_spec=MIN_CLUSTER_SPEC)
+    @parametrize(num_active_partitions=1, txn_per_client=100, num_clients=2, interval=600, timeout=360,
+                 interruption_duration=10, num_interruptions=3, delay_between_interruptions=5)
```

Comment:
nit: `interrupt_duration` instead of `interruption_duration` may be?? what do you think?